compiler/next/include/chpl/resolution/resolution-types.h

```diff
@@ -433,6 +454,18 @@ class TypedFnSignature {
   bool operator!=(const TypedFnSignature& other) const {
     return !(*this == other);
   }
+  static bool update(owned<TypedFnSignature>& keep,
+                     owned<TypedFnSignature>& addin) {
+    return defaultUpdateOwned(keep, addin);
+  }
+  void mark(Context* context) const {
+    context->markUnownedPointer(untypedSignature_);
+    for (auto const &elt : formalTypes_) {
+      elt.mark(context);
+    }
+    context->markUnownedPointer(instantiatedFrom_);
+    context->markUnownedPointer(parentFn_);
```

Comment:
is the order of marking important here? Only asking b/c not all the `markUnownedPointer` calls are grouped together

Reply:
No the order isn't important - here I just used the field declaration order.